src/traces/scattermapbox/hover.js

```diff
@@ -19,6 +19,8 @@ module.exports = function hoverPoints(pointData, xval, yval) {
         xa = pointData.xa,
         ya = pointData.ya;
 
+    if(cd[0].placeholder) return;
```

Comment:
is there another name we can use beside `placeholder`,  a `placeholder` for `_____`? How about `emptytrace`?.